docs/static/openapi.yml

```diff
@@ -471,6 +471,7 @@ paths:
             type: object
```

Comment:
@AustinoBombino there are a lot of changes that seem to have happened in this file that will be very difficult to review, do you mind explain in providing a summary of what changed here thanks

Reply:
I think these are just changes that Ignite generated for the openapi docs

Reply:
@AustinoBombino oh gotcha in that case we might need to make sure we are using the same version of ignite, here is mine: 

```
$ ignite version
Ignite CLI version:		v0.26.1
Ignite CLI build date:		2023-01-24T18:50:18Z
Ignite CLI source hash:		d2a2e287c176ba010557f0b8fbaa198cdeee928d
Ignite CLI config version:	v1
Cosmos SDK version:		v0.46.7
Your OS:			linux
Your arch:			amd64
Your Node.js version:		v19.3.0
Your go version:		go version go1.19.4 linux/amd64
[...]
```

Reply:
@cosmic-vagabond 
```
$ignite version
Ignite CLI version:             v0.26.1
Ignite CLI build date:          2023-01-24T18:50:18Z
Ignite CLI source hash:         d2a2e287c176ba010557f0b8fbaa198cdeee928d
Ignite CLI config version:      v1
Cosmos SDK version:             v0.46.7
Your OS:                        linux
Your arch:                      amd64
Your Node.js version:           v18.9.0
Your go version:                go version go1.19 linux/amd64
```